addons/ailogistics/functions/fnc_logisticWaypoints.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Assign the waypoints for the logistical vehicles.
+
+    Parameters:
+        0: _group <GROUP>
+
+    Return value:
+        Nothing
+*/
+
+params ["_module", "_group"];
+
+private _destinationModule = (synchronizedObjects _module) select {_x isKindOf "uksf_ailogistics_logisticsDestination"};
```

Comment:
Use the macro version instead of magic strings.
`QGVAR(logisticsDestination)`
and
`QGVAR(logisticsPicketGroup)`